src/Validation/FieldTextPolicyValidator.php

```diff
@@ -15,7 +15,7 @@ class FieldTextPolicyValidator {
 	public function __construct( private readonly TextPolicyValidator $textPolicyValidator ) {
 	}
 
-	public function validate( $value ): ValidationResult {	// @codingStandardsIgnoreLine
+	public function validate( string $value ): ValidationResult {	// @codingStandardsIgnoreLine
```

Comment:
This class is entirely unused outside of its test. Please delete it and its test.